### PR TITLE
fix(SynchronizableRenderWindow): Expose some missing functionality

### PR DESCRIPTION
**Pull request by scottwittenburg**

### CODE REVIEW

File: Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js [hunk 1]
```diff
@@ -290,8 +289,8 @@ function polydataUpdater(instance, state, context) {
       .then(
         (values) => {
           const array = vtkDataArray.newInstance(Object.assign({ values }, arrayMetadata));
-          // console.log('array', array.getName());
-          arraysToBind.push([instance.get(arrayMetadata.location)[arrayMetadata.location].addArray, [array]]);
+          const regMethod = arrayMetadata.registrationMethod ? arrayMetadata.registrationMethod : 'addArray';
```
Comment (jourdain) [inline Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js]: I don't know if we want the same name as [here](https://github.com/Kitware/vtk-js/blob/master/Sources/IO/Core/HttpDataSetReader/index.js#L62)

I mean, I like the explicit part of `registrationMethod` vs `registration`, but just wondering.

**Comment by scottwittenburg** [comment]
@jourdain The most recent commit here, along with the VTK MR I just pinged you about, is resulting in the shading we expect in the client.

screen shot 2017-05-09 at 11 22 37 am

File: Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js (hunk 1 above)
Comment (scottwittenburg) [inline Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js]: I'd almost rather change that dataset reader to say `registrationMethod`, but up to you...

Comment (jourdain) [inline Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js]: If you change the dataset reader, you will have to fix the data as well and the python script that `may` generate that attribute.

**Comment by scottwittenburg** [comment]
@jourdain I think this could be good now.

**Pull request merged**